nix/about.tt

```diff
@@ -188,18 +188,18 @@ href="https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/
 Nix expression</a>:</p>
 
 <pre>
-<span class="muted">$</span> nix-shell '&lt;nixpkgs>' -A pan
+<span class="command">nix-shell '&lt;nixpkgs>' -A pan</span>
```

Comment:
Since this appropriately uses `&lt;` for `<`, can it be updated to _also_ use `&gt;` for `>` (for consistency)?

```html
- <span class="command">nix-shell '&lt;nixpkgs>' -A pan</span>
+ <span class="command">nix-shell '&lt;nixpkgs&gt;' -A pan</span>
```